com.civicrm.hrjobroles/CRM/Hrjobroles/Import/Parser/HrJobRoles.php

```diff
@@ -147,6 +147,7 @@ function summary(&$values) {
     $session = CRM_Core_Session::singleton();
     $dateType = $session->get('dateTypes');
 
+    /** @var object $contractDetails */
```

Comment:
Wouldn't it make more sense to add the type hint to the checkContract (which is wrongly saying it returns an array instead of an object)?